v2/tools/generator/internal/armconversion/convert_to_arm_function_builder.go

```diff
@@ -738,11 +824,7 @@ func (builder *convertToARMBuilder) convertComplexTypeNameProperty(conversionBui
 
 	if !destinationType.PackageReference.Equals(conversionBuilder.CodeGenerationContext.CurrentPackage()) {
 		// needs to be qualified
-		packageName, err := conversionBuilder.CodeGenerationContext.GetImportedPackageName(destinationType.PackageReference)
-		if err != nil {
-			panic(err)
-		}
-
+		packageName := conversionBuilder.CodeGenerationContext.MustGetImportedPackageName(destinationType.PackageReference)
```

Comment:
This can panic. If you want error returns here... shouldn't this use the error-returning flavor like it did before, but instead return an error?

Reply:
Changed.